support-services/src/main/scala/com/gu/support/abtests/BenefitsTest.scala

```diff
@@ -0,0 +1,35 @@
+package com.gu.support.abtests
+
+import com.gu.i18n.Currency
+import com.gu.i18n.Currency.{CAD, EUR, GBP, NZD, USD}
+import com.gu.support.acquisitions.AbTest
+import com.gu.support.workers.{Annual, BillingPeriod, DigitalPack, Monthly}
+
+object BenefitsTest {
+  def isValidBenefitsTestPurchase(product: DigitalPack, maybeAbTests: Option[Set[AbTest]]) =
```

Comment:
what does it mean if `maybeAbTests` is `None`?

Reply:
From the point of view of this method it just means that they are not in the test variant for this test so this is not a valid benefits test purchase